scipy/sparse/linalg/eigen/_svds.py

```diff
@@ -1,17 +1,17 @@
-__all__ = ['svds']
-
 import numpy as np
 
 from .arpack import _arpack
-arpack_int = _arpack.timing.nbx.dtype
 from . import eigsh
-from ._svds_doc import _svds_arpack_doc, _svds_lobpcg_doc
```

Comment:
PEP8 doesn't like imported but unused, so I made the required adjustements to the `.rst` files above.
The other two are PEP8 to keep imports at the top of a file.